packages/app-store/exchangecalendar/lib/CalendarService.ts

```diff
@@ -198,7 +198,7 @@ export default class ExchangeCalendarService implements Calendar {
   }
 
   private getExchangeService(): ExchangeService {
-    const service: ExchangeService = new ExchangeService();
+    const service: ExchangeService = new ExchangeService(this.payload.exchangeVersion);
```

Comment:
```suggestion
    const service: ExchangeService = new ExchangeService(ExchangeVersion[this.payload.exchangeVersion]);
```

@copoer thank you so much for your contribution. When initiating the calendar service I noticed that the number was being passed and not the Exchange version.

Reply:
The ExchangeService constructor is expecting a enum not a string. This does not need to be added.
https://github.com/gautamsi/ews-javascript-api/blob/9488cdf389f16a7daa8a17299ab195a487b9b30f/typings/ExchangeWebService.d.ts#L241